src/components/TimeseriesChart/stories/basic.story.mdx

```diff
@@ -408,3 +410,46 @@ const series = [
     />
   </Story>
 </Preview>
+
+### Define Y-axis Subdomain
+
+```jsx
+const ySubDomains = { 123: [-130, 130] };
+```
+
+<Preview>
+  <Story name="Define Y-axis Subdomain" >
+    <TimeseriesChart
+      timeseriesIds={[123]}
+      ySubDomains = {ySubDomains}
+    />
+  </Story>
+</Preview>
+
+### Define Y-axis Subdomain For Series
+
+```jsx
+import { AxisDisplayMode } from '@cognite/griff-react';
+
+const seriesWithCustomYdomain = [
```

Comment:
can we add here exact same structure that we're importing for story example (`yAxisDisplayMode: AxisDisplayMode.ALL` is missed)